### ISSUE TITLE
Error EXR: 'deepscanline'. warning: 'deepImageState', 'idmanifest'.

### ISSUE THREAD

**Issue by TodicaIonut** [mrdoob/three.js#28055]
### Description

warning:
THREE.EXRLoader: Skipped unknown header attribute type 'deepImageState'. [here](https://openexr.com/en/latest/TechnicalIntroduction.html#multi-part-and-deep-data-attributes).
THREE.EXRLoader: Skipped unknown header attribute type 'idmanifest'.

Error message:
type: "deepscanline" [here](https://openexr.com/en/latest/TechnicalIntroduction.html#multi-part-and-deep-data-attributes)
Error: THREE.EXRLoader: Provided file is currently unsupported.

### Reproduction steps

1. warning deepImageState
2. warning idmanifest
3. Error deepscanline

### Code

```js
type : "deepscanline"
```

### Live example

- [gainmap-creator.monogrid.com](https://gainmap-creator.monogrid.com/)

### Screenshots

chrome:
![Untitled29](https://github.com/mrdoob/three.js/assets/156521906/0ec77fa7-e666-4cd0-9a68-494900d60992)

mrv2:
![Untitled30](https://github.com/mrdoob/three.js/assets/156521906/45e83967-d522-40db-b14f-22fa72a00b32)

### Version

r164

### Device

Desktop

### Browser

Chrome

### OS

Windows

**Comment by TodicaIonut** [comment]
[OpenEXR spec](https://openexr.com/en/latest/TechnicalIntroduction.html#multi-part-and-deep-data-attributes), deepscanline is defined like so:

> Deep scan line images: indicated by a type attribute of deepscanline.

adding support for deepscanline is feasible?
adding support for deepImageState is feasible?
adding support for idmanifest is feasible?
[42.deep.exr](https://github.com/mrdoob/three.js/files/14855241/42.deep.zip)
[64.deep.exr](https://github.com/mrdoob/three.js/files/14855243/64.deep.zip)
[multivariate.deep.exr](https://github.com/mrdoob/three.js/files/14855245/multivariate.deep.zip)
[objectid.deep.exr](https://github.com/mrdoob/three.js/files/14855249/objectid.deep.zip)
cc @sciecode 
![Untitled28](https://github.com/mrdoob/three.js/assets/156521906/1f4e1a65-eb3f-4ae4-b38d-d43a78613c54)

**Comment by TodicaIonut** [comment]
[Deep EXR Test](https://github.com/TodicaIonut/Test-EXR) cc @sciecode

**Comment by TodicaIonut** [comment]
Deep Data Header Attributes:

> Must be set to `deepscanline` or `deeptile`.

> Deep scan line images are indicated by a type attribute of “`deepscanline`”. Each chunk of deep scan line data is a single scan line of data. The data in each chunk is laid out as follows:

> Tiled images are indicated by a type attribute of “`deeptile`”. Each chunk of deep tile data is a single tile. The data in each chunk is laid out as follows:

adding support for deep images to `EXRLoader`.

@Mugen87 Not Support for deep images

**Comment by TodicaIonut** [comment]
EXRLoader: Deep Data, deepFormat **true** not work.

```
deepFormat: true
longName: false
multiPart: true
singleTile: false
type: "deepscanline"
```

```
if ( ( spec & ~ 0x06 ) != 0 ) { // unsupported deep-image, multi-part
```
/cc @mrdoob @WestLangley

**Comment by WestLangley** [comment]
@TodicaIonut Yes, tile-based data, deep image, and multi-part data are not yet supported by `EXRLoader`.

What is the compelling reason you need this? Can you please explain your use case in detail?
